desktop/core/src/desktop/js/apps/editor/components/resultChart/ko.resultChart.js

```diff
@@ -863,8 +863,11 @@ class ResultChart extends DisposableComponent {
   }
 
   prepopulateChart() {
-    const type = this.chartType();
-    hueAnalytics.log('notebook', 'chart/' + type);
+
+    if (!window.ENABLE_NOTEBOOK_2) {
```

Comment:
This is to prevent the log from being executed each time the page renders in v2 of the notebook